meta-lmp-base/recipes-support/lmp-el2go-auto-register/lmp-el2go-auto-register.bb

```diff
@@ -0,0 +1,27 @@
+SUMMARY = "A systemd oneshot helper to auto register a device using EdgeLock2GO"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+RDEPENDS:${PN} += "plug-and-trust-seteec python3-core python3-plug-and-trust-ssscli opensc"
+
+SRC_URI = " \
+	file://lmp-el2go-auto-register.service \
+	file://lmp-el2go-auto-register \
+	file://default.env \
+	file://root.crt \
```

Comment:
These two are empty so we can achieve the same in `do_install` using the `touch` command and keep meta-lmp more clean

Reply:
I stole from lmp-device-auto-register, so Ricardo can probably explain better. But,this approach is designed to let people override the file content in meta-subscriber-overrides. This recipe on its own is useless, the two empty files must be overridden with correct content. I don't think the "touch" approach would do that?

Reply:
Yeah, the idea to have an empty file by default is that we can replace it by simply adding a file that has proper content under meta-subscriber (without having to add logic in a bbappend, besides FILESEXTRAPATHS).